caravel/config.py

```diff
@@ -206,6 +206,9 @@ class CeleryConfig(object):
 # The db id here results in selecting this one as a default in SQL Lab
 DEFAULT_DB_ID = None
 
+# Timeout duration for SQL Lab synchronous queries
```

Comment:
s/Timeout duration for SQL Lab synchronous queries/Timeout duration for SQL Lab synchronous queries in seconds.